### ISSUE TITLE
Feature request: remember position

### ISSUE THREAD

**Issue by ulno** [gphalkes/tilde#21]
It would be nice if tilde could remember the last position (line and column) editing a file. So If you re-open a file, you are back at the line you were editing before (this of course should be overwritten by specifying a edit position at the command line).

**Comment by ulno** [comment]
Maybe it would be enough to give tilde an option to just dump a list of all open files and the cursor position in them on closing to stdout or stderr?

**Comment by gphalkes** [comment]
This has been implemented in commit https://github.com/gphalkes/tilde/commit/9a74b20fcebf9572d25d8290f68a3a71fc6025e4 and will be part of the next Tilde release. 

However, I'm not 100% happy yet with the functionality: although the cursor position is restored, the view position is not restored. This means that although the cursor is visible after loading, it will likely be on the last visible line, rather than on the screen position where it was before. Of course this can't be done anyway if the screen size is different, but if the screen size is the same, the result of closing and then opening a file should be a no-op. So I'll leave the bug open to remind me to implement this part as well.

**Comment by Eucyon** [comment]
I don't get it running.
Version: 1.0.3
~/.config/tilde/config:
```config_version = 1
%terminals {
	name = "xterm-256color"
	key_timeout = 100
	color = false
}
make_backup = true
hide_menubar = false
parse_file_positions = true
disable_primary_selection_over_ssh = false
restore_cursor_position = true
save_recent_files = true
```
What's wrong?
Thank you!

**Comment by gphalkes** [comment]
Sorry, the feature has not been released yet. The 1.0.2 and 1.0.3 releases only included fixes for important bugs. You will have to wait a little while longer to use the feature.

**Comment by Eucyon** [comment]
Thank you, I am looking forward.

**Comment by gphalkes** [comment]
This has now been released as part of version 1.1.0
